gdc_ng_models/models/study.py

```diff
@@ -0,0 +1,134 @@
+"""Data models to describe studies.
+
+Studies will be used for the single-study controlled-access feature in gdcapi.
+These models provide a way to associate programs and projects to studies.
+"""
+
+from sqlalchemy.ext import declarative
+from sqlalchemy.sql import schema, sqltypes
+
+from gdc_ng_models.models import audit
+
+Base = declarative.declarative_base()
+
+
+class Study(Base, audit.AuditColumnsMixin):
+    """A study for use with single-study controlled-access.
+
+    Attributes:
+        id: A unique identifier for the study.
+        name: A unique, informational name for the study.
+        created_date: The date and time when the record is created.
+        updated_date: The date and time when the record is updated.
+    """
+
+    __tablename__ = "study"
```

Comment:
should we prefix the table name?

Reply:
If we serve multiple applications out of the same database, then we should definitely prefix every table.  It's the only safe way to control the table namespace in that scenario. We're not doing that, are we?

Is there another reason you think we need prefixes?

Reply:
at least like an indication that these tables are of the same "group" and to not get confused with something else like `node_study` (if we get that table at some point for example).

Reply:
suggestions for a prefix?  If we use a prefix, should the same prefix apply to the tables that @ProfOak is creating?

Reply:
maybe something like `dave_`? probably should apply to both?

Reply:
gdcapi exists independently of DAVE.  `api_` seems a little too generic, unless we want to start prefixing all of our tables with that.  thoughts?

Reply:
our current "conventions" (idk how to call that 🤣 ) is to have related tables share prefix. e.g. all dictionary related tables are `node_*` and `edge_*`, for QC it's `qc_*`, so I was just thinking that since these new models are (mostly) related to visualization feature, maybe we can use `dave_*`. or we can just ignore what I said and maybe `study` is ok in this case 🤷‍♂️